.pre-commit-config.yaml

```diff
@@ -0,0 +1,60 @@
+repos:
```

Comment:
@swanderz : I copied something from a cookiecutter project we have, I would suggest you go through them and add comments about which one to use.